app/views/styleguide/pages_parent_category_page.html.erb

```diff
@@ -1,16 +1,62 @@
-<div class="l-main">
-  <div class="l-main__row">
-    <main class="l-main__cell-single" role="main">
-      <h1>Leaving school or college</h1>
+<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+  <symbol id="svg-icon--plus" viewBox="0 0 22.354 22.354">
+    <polygon points="8.61,22.354 13.744,22.354 13.744,13.744 22.354,13.744 22.354,8.611 13.744,8.611 13.744,0 8.61,0 8.61,8.611 0,8.611 0,13.744 8.61,13.744"/>
+  </symbol>
+</svg>
+<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+  <symbol id="svg-icon--minus" viewBox="0 0 22.354 22.354">
+    <rect y="8.61" width="22.354" height="5.133" />
+  </symbol>
+</svg>
 
-      <p class="intro">Help and advice on staying on in education, moving into work and paying your own
-        way as a young adult</p>
+<div class="l-main editorial">
+  <div class="l-main__row">
+    <style>
+      .l-category--image {
+        background-image: url(<%= image_url('atm_en.png') %>);
+      }
 
+      @media only screen and (-webkit-min-device-pixel-ratio: 2),
+      only screen and (min--moz-device-pixel-ratio: 2),
+      only screen and (-o-min-device-pixel-ratio: #{(2*10)}/10),
+      only screen and (min-device-pixel-ratio: 2),
+      only screen and (min-resolution: #{(2*96)}dpi),
+      only screen and (min-resolution: #{2}dppx) {
+        .l-category--image {
+          background-image: url(<%= image_url('atm_en.png') %>);
+        }
+      }
+    </style>
+    <main class="l-main__cell-full l-category l-category--image" role="main" data-parent="">
+      <div class="l-category__top">
+        <div class="l-category__top__content">
+          <h1 aria-level="1" class="l-category__heading" role="heading">Debt and borrowing</h1>
+          <span class="l-category__top__background"></span>
+          <p class="l-category__top__intro">Before you sign up for a credit card, bank loan or store card, or add to an existing card or loan it makes sense to think about whether you really need to borrow money.</p>
+        </div>
+        <h2 aria-level="2" class="l-category__top__link-list-heading" role="heading">How can we help?</h2>
+        <ul class="link-list">
+          <li class="link-list__item">
+            <a class="link-list__item__link" href="/en/">Before you borrow</a>
+          </li>
+          <li class="link-list__item">
+            <a class="link-list__item__link" href="/en/">Taking control of debt</a>
+          </li>
+          <li class="link-list__item">
+            <a class="link-list__item__link" href="/en/">Mortgage and rent problems</a>
+          </li>
+          <li class="link-list__item">
+            <a class="link-list__item__link" href="/en/">Help with loans</a>
+          </li>
+          <li class="link-list__item">
+            <a class="link-list__item__link" href="/en/">Credit cards and credit ratings</a>
+          </li>
+        </ul>
+      </div>
```

Comment:
Oops, totally missed that this was part of the styleguide. In that case many of these comments will be overly-picky I suspect. Sorry!